include/aikido/constraint/SequentialSampleable.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef AIKIDO_CONSTRAINT_SEQUENTIALSAMPLEABLE_HPP_
+#define AIKIDO_CONSTRAINT_SEQUENTIALSAMPLEABLE_HPP_
+
+#include "aikido/constraint/Sampleable.hpp"
+
+namespace aikido {
+namespace constraint {
+
+/// Sampleable supporting a composition of multiple sampleables which
```

Comment:
I think this is confusing. Here's my stab at it:

`Sampleable` that wraps a sequence of `Sampleable`s. Exhausts the current `Sampleable` before sampling from the next `Sampleable` in the sequence.